tools/Import/Importer.cs

```diff
@@ -0,0 +1,358 @@
+                }
+                catch (Exception e)
+                {
+                    if (IsNetworkError(e))
+                    {
+                        Console.WriteLine($"Retries={retries} blob={blob.Name} error={e.Message}");
+                        Thread.Sleep(1000);
+                        retries++;
+                        goto retry;
+                    }
+
+                    throw;
+                }
+
+                results.AddRange(resInt);
+                lines += resInt.Count;
+                Interlocked.Add(ref totalReads, resInt.Count);
+            }
+
+            if (Interlocked.Decrement(ref readers) == 0)
+            {
+                swReads.Stop();
+            }
+
+            Console.WriteLine($"{logPrefix}: Completed reads. Total={lines} secs={(int)sw.Elapsed.TotalSeconds} speed={(int)(lines / sw.Elapsed.TotalSeconds)} lines/sec");
+            return results;
+        }
+
+        private static BlobContainerClient GetContainer(string connectionString, string containerName)
+        {
+            try
+            {
+                var blobServiceClient = new BlobServiceClient(connectionString);
+                var blobContainerClient = blobServiceClient.GetBlobContainerClient(containerName);
+
+                if (!blobContainerClient.Exists())
+                {
+                    Console.WriteLine($"Storage container {containerName} not found.");
+                    return null;
+                }
+
+                return blobContainerClient;
+            }
+            catch
+            {
+                Console.WriteLine($"Unable to parse stroage reference or connect to storage account {connectionString}.");
+                throw;
+            }
+        }
+
+        private static void PutResource(string jsonString, IndexIncrementor incrementor)
+        {
+            ParseJson(jsonString, out var resourceType, out var resourceId);
+            using var content = new StringContent(jsonString, Encoding.UTF8, "application/json");
+            var maxRetries = MaxRetries;
+            var retries = 0;
+            var networkError = false;
+            var bad = false;
+            string endpoint;
+            do
+            {
+                endpoint = endpoints[incrementor.Next()];
+                var uri = new Uri(endpoint + "/" + resourceType + "/" + resourceId);
+                bad = false;
+                Interlocked.Increment(ref epCalls);
+                try
+                {
+                    Thread.Sleep(40);
+                    var response = HttpClient.PutAsync(uri, content).Result;
+                    switch (response.StatusCode)
+                    {
+                        case HttpStatusCode.OK:
+                        case HttpStatusCode.Created:
+                            break;
+                        default:
+                            bad = true;
+                            var statusString = response.StatusCode.ToString();
```

Comment:
Perhaps just use the enum?
`status != HttpStatusCode.BadGateway`?

Reply:
Right.